ckanext/timeline/public/fanstatic/hc.js

```diff
@@ -394,6 +395,37 @@ $(function () {
             else if (format == 'zero') {
                 c = Number(val);
             }
+            else if (format == 'dt' || format == 'dt2zero') {
+                var m = 0;
+                if (moment.utc(val.trim(), moment.ISO_8601, true).isValid()) {
+                    m = moment.utc(val.trim(), moment.ISO_8601, true);
+                    /* Pick the end of year for end boxes. */
+                    if (jquery.is('#end, #ext_timeline_end')) {
```

Comment:
Don't use IDs directly. Use the end_box and end_box_hidden variables, if possible.

Reply:
Ok. Changed.